OmiseSDK/Compatibility/OMSConstants.h

```diff
@@ -22,6 +22,7 @@ extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValuePromptPay;
 extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValuePayNow;
 extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueTrueMoney;
 extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValuePointsCiti;
+extern OMSSourceTypeValue const _Nonnull OMSSourceTypeValueFPX;
```

Comment:
Can you rename this to `OMSSourceTypeValueFpx`? If you use only capitalized letters for `FPX` we will get `.FPX` in swift and it's not common to use capitalized letters in a swift enum. And since the SDK is swift-first we should try  to follow the swift style.